### DISCUSSION TITLE
Add free and premium option for users

### DISCUSSION THREAD

**Discussion by analogllp** [danny-avila/LibreChat#249] [Feature Requests & Suggestions]
It can help to provide gpt3.5 turbo for free and limited access for gpt4 to some users who need it and limits also for gpt4

**Comment by danny-avila** [comment]
In the interest of making an open source project, we won't be adding business features. Maybe admin dashboard/user permissions will come later down the line, but that is out of scope for now. You are free to fork the project as you wish.
